ngraph/test/files/paddlepaddle/gen_scripts/generate_pow.py

```diff
@@ -13,7 +13,7 @@ def pdpd_pow(name : str, x, y, data_type):
     with pdpd.static.program_guard(pdpd.static.Program(), pdpd.static.Program()):
         node_x = pdpd.static.data(name='x', shape=x.shape, dtype = data_type)
         out = pdpd.fluid.layers.pow(node_x, y, name = 'pow')
-
+        out = pdpd.cast(out, "float32")
```

Comment:
this 'cast' could hide the problems with 'scale that we had (that result was always 'fp32'). It may be still better to make appropriate 'test_case.run' for int32 cases

Reply:
Enable int32 test in FuzzyTest. Fix this in 
f59d261